src/components/Shop/CategoryBar/CategoryButton.tsx

```diff
@@ -10,16 +15,22 @@ type CategoryButtonProps = {
 const CategoryButton: FC<CategoryButtonProps> = ({
   category,
 }: CategoryButtonProps) => {
-  const { dispatch } = useContext(StateContext);
+  const { state, dispatch } = useContext(StateContext);
   const onClick = () => dispatch(setCategory(category));
 
-  return <Button onClick={onClick}> {category} </Button>;
+  return (
+    <Button
+      style={category == state.category ? selectedStyle : notSelectedStyle}
```

Comment:
Instead of creating two objects for this, I think
`style={{ backgroundColor: category == state.category ? OnlineOrange : OnlineBlue}}`
works just as well, and might be easier to read.